examples/mixture_models/dependent_density_regression.ipynb

```diff
@@ -2,7 +2,16 @@
  "cells": [
```

Comment:
<p>This is fine, but I think you could do the handling on <code>w</code> directly with <code>xarray</code>:  http://xarray.pydata.org/en/stable/computation.html<span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
ah, thanks! OK I'll do that next time